cruise-control-client/cruisecontrolclient/client/Responder.py

```diff
@@ -93,8 +96,9 @@ def json_or_text_guesser():
             else:
                 # Guess about whether this version of cruise-control supports 202
                 if "Cruise-Control-Version" in response.headers:
+                    non_decimal = re.compile(r'[^\d.]+')
```

Comment:
This is a nit, but suggest a comment like  
```python
# define a regex for extracting only leading digits and periods from the Cruise-Control-Version
```

Reply:
Otherwise, I agree this looks good

Reply:
I'll add this. I'll use your exact comment.